1 - identify organisations in the corpus/preprocessing.py

```diff
@@ -0,0 +1,268 @@
+from text_preprocessing import preprocess_text
+from text_preprocessing import to_lower, remove_number, remove_itemized_bullet_and_numbering, remove_special_character, remove_punctuation, remove_whitespace, normalize_unicode, check_spelling, remove_url, remove_email, remove_phone_number, remove_ssn, remove_whitespace
+from nltk.tokenize import word_tokenize, sent_tokenize
+from nltk.corpus import stopwords
+from nltk.stem import WordNetLemmatizer
+import re
+import pandas as pd
+import csv
+import os.path
+import json
+import string
+
+datafr = pd.read_csv('../data-files/master-ner-results.csv')
```

Comment:
Here, the script tries to load a file that has not been generated yet!

Reply:
Yep, fixed this by moving it to the function where the file is needed.